packages/sdk/src/modules/loggerModule/index.ts

```diff
@@ -1,14 +1,28 @@
 import { LoggerFactory, LoggerType } from "@vue-storefront/logger";
 import type { Module } from "../../types";
 import type { LoggerModuleConfig } from "./types";
+import { injectMetadata } from "./injectMetadata";
+import { isInvalidConfig } from "./utils";
 
 export const loggerModule = (options?: LoggerModuleConfig) => {
   const logger =
     options?.handler ?? LoggerFactory.create(LoggerType.ConsolaGcp, options);
 
+  if (isInvalidConfig(options)) {
+    console.warn(
+      "Both handler and options are provided to logger's options. Using handler, options will be ignored."
```

Comment:
Please, use this file as a reference: we want to have more verbose message with a clear steps of what to do and where to look
https://github.com/vuestorefront/vue-storefront/blob/553dd5144c9d0fdeb6e2c32b1783b85a6efd10aa/packages/middleware/src/apiClientFactory/applyContextToApi.ts

Reply:
Added, @bartoszherba please take a look.